internal/plugins/ansible/templates/roles/handlers_main.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package roles
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/constants"
+)
+
+const handlersMainPath = "handlers" + constants.FilePathSep + "main.yml"
+
+type HandlersMain struct {
+	file.TemplateMixin
+	file.ResourceMixin
+}
+
+// SetTemplateDefaults implements input.Template
+func (f *HandlersMain) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = filepath.Join(constants.RolesDir, "%[kind]", handlersMainPath)
+		f.Path = f.Resource.Replacer().Replace(f.Path)
+	}
```

Comment:
same ; https://github.com/operator-framework/operator-sdk/pull/3488/files/771dbcef0117180f1de0a7687fc3b3c9a04ced01#r459194209

Reply:
resolving these to make parsing the comments easier. we can discuss on the main comment.